packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx

```diff
@@ -12,18 +12,54 @@ const meta: ComponentMeta<typeof InlineLabel> = {
       page: mdx,
     },
   },
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    tooltip: {
+      control: 'text',
+    },
+    width: {
+      control: 'text',
+    },
+    transparent: {
+      control: 'boolean',
+    },
+    interactive: {
+      control: 'boolean',
+    },
```

Comment:
We don't need to include these as Storybook can guess what the controls should be based on the types of the props 😄